tests/main/views/test_suppliers.py

```diff
@@ -1542,6 +1605,78 @@ def test_get_supplier_framework_info_with_non_existent_supplier(self, supplier_f
 
         assert response.status_code == 404
 
+    def test_set_application_company_details_confirmed_updates_declaration(self, supplier_framework):
```

Comment:
Maybe this should go in the class below.

Reply:
Yep 👍